partiql-planner/src/test/kotlin/org/partiql/planner/util/Utils.kt

```diff
@@ -32,6 +33,55 @@ val allSupportedType = StaticType.ALL_TYPES.filterNot {
     it == StaticType.GRAPH || it is NullType || it is MissingType
 }
 
+val allDatePType = setOf(PType.date())
+
+val allTimePType = setOf(
+    PType.time(6), // TODO: Precision
+    PType.timez(6), // TODO: Precision
+)
+
+val allTimeStampPType = setOf(
+    PType.timestamp(6), // TODO: Precision
+    PType.timestampz(6), // TODO: Precision
+)
+
+val allDateTimePType = allDatePType + allTimePType + allTimeStampPType
+
+val allCharStringPType = setOf(
+    PType.character(256), // TODO: Length
+    PType.varchar(256), // TODO: Length
+    PType.string(),
+    PType.clob(Int.MAX_VALUE), // TODO: Length
+)
+
+val allBinaryPType = setOf(
+    PType.blob(Int.MAX_VALUE), // TODO: Length
```

Comment:
Will these TODOs for precision/length be addressed in this PR or are they tracked in some other issue?

Reply:
https://github.com/partiql/partiql-lang-kotlin/issues/1649